triton/stream_reader.go

```diff
@@ -37,13 +37,19 @@ func (msr *multiShardStreamReader) Checkpoint() (err error) {
 	return
```

Comment:
Actually it looks like ShardStreamReader uses LastSequenceNumber to initialize the iteration position